### DISCUSSION TITLE
(Suggestion) Add a Set List Version Number and Display it on the Game List

### DISCUSSION THREAD

**Discussion by dude1286** [RetroAchievements/RAWeb#796] [Ideas]
Ever since the change to how the points are distributed across the games and achievements themselves, It has been difficult to determine what games are currently being worked on and whether or not a list has achievements across the entire game. Adding a versioning number to the lists could help players determine which game to pick next without having to disect the list itself to see if it goes through the entire game. This would also help some players decide to pick up a game they never played before and wouldn't know by looking at the list whether or not it covers the game from start to finish.

**Comment by Keltron3030** [comment]
Revisions and/or a proper change log are planned for the next implementation of the website (V2).  There's no time frame on that but in the meantime the best place to check if a set is in the works is on the forum page.  Developers are required to make a post with their revision plan so if you see an open-ended post that's your indication that there's a work in progress.  Many revisions are just a quick small change but the larger ones that take more time should be obvious by the depth and scope of the revision plan.  Developers are almost certain to give an update when they've completed their revision.

**Comment by dude1286** [comment]
Ok, I kind of left the community about 5 years ago and have only been in and out completing a list here and there. When the 400 point limit was removed and the achievements themselves started getting their points regulate, it became harder to determine if a list is (at the moment) complete or not. I can also see a lot of games that were incomplete when I left haven't had any changes since I left outside of maybe the points themselves, so it made me go into the list just to see if a list went from the beginning to the end of the game, which required prior knowledge. Might be a good idea to implement a smaller change that can help players if Website2.0 doesn't have a time frame.

**Comment by Jamiras** [comment]
See also #688 - a set could be marked as Complete when it's complete.

**Comment by dude1286** [comment]
What is in that issue does look nicer.
